src/object/srv_obj_migrate.c

```diff
@@ -1762,7 +1762,7 @@ migrate_one_epoch_object(daos_handle_t oh, daos_epoch_range_t *epr,
 		num = KDS_NUM;
 		daos_anchor_set_flags(&dkey_anchor,
 				      DIOF_TO_LEADER | DIOF_WITH_SPEC_EPOCH |
-				      DIOF_TO_SPEC_SHARD);
+				      DIOF_TO_SPEC_GROUP);
```

Comment:
for EC, seems should use SPEC_SHARD, @wangdi1 ?
although in this patch, the SPEC_GROUP keep the same behavior as SPEC_SHARD so probably fine, but seems some other change needed later (assume Di will do?)